tests/integration_tests/test_botorch.py

```diff
@@ -78,9 +79,9 @@ def candidates_func(
         train_obj: torch.Tensor,
         train_con: Optional[torch.Tensor],
         bounds: torch.Tensor,
-    ) -> torch.Tensor:
+    ) -> Any:
```

Comment:
`List[float]` sounds more accurate than `Any`. What do you think?
```suggestion
    ) -> List[float]:
```

Reply:
This is a test case for invalid type, and making it `List[float]` would have to introduce a `# type: ignore`.

Reply:
Right. Somehow my local mypy was happy with `List[float]`. Thank you for your explanation!

Reply:
Could you add a comment to explain why we put `Any` here? Apart from it, LGTM.

Reply:
I added a comment. PTAL.